tools/src/main/java/org/apache/kafka/trogdor/workload/ConnectionStressWorker.java

```diff
@@ -99,19 +99,33 @@ public void start(Platform platform, WorkerStatusTracker status,
         log.info("{}: Activating ConnectionStressWorker with {}", id, spec);
         this.doneFuture = doneFuture;
         this.status = status;
-        this.totalConnections = 0;
-        this.totalFailedConnections  = 0;
-        this.startTimeMs = TIME.milliseconds();
+        synchronized (ConnectionStressWorker.this) {
```

Comment:
Had to add this here otherwise we got
```

IS | Inconsistent synchronization of org.apache.kafka.trogdor.workload.ConnectionStressWorker.nextReportTime; locked 66% of time
-- | --
  | Bug type IS2_INCONSISTENT_SYNC (click for details) In class org.apache.kafka.trogdor.workload.ConnectionStressWorkerField org.apache.kafka.trogdor.workload.ConnectionStressWorker.nextReportTimeSynchronized 66% of the timeUnsynchronized access at ConnectionStressWorker.java:[line 107]Synchronized access at ConnectionStressWorker.java:[line 124]Synchronized access at ConnectionStressWorker.java:[line 162]
IS | Inconsistent synchronization of org.apache.kafka.trogdor.workload.ConnectionStressWorker.startTimeMs; locked 50% of time
  | Bug type IS2_INCONSISTENT_SYNC (click for details) In class org.apache.kafka.trogdor.workload.ConnectionStressWorkerField org.apache.kafka.trogdor.workload.ConnectionStressWorker.startTimeMsSynchronized 50% of the timeUnsynchronized access at ConnectionStressWorker.java:[line 104]Synchronized access at ConnectionStressWorker.java:[line 120]
IS | Inconsistent synchronization of org.apache.kafka.trogdor.workload.ConnectionStressWorker.totalConnections; locked 66% of time
  | Bug type IS2_INCONSISTENT_SYNC (click for details) In class org.apache.kafka.trogdor.workload.ConnectionStressWorkerField org.apache.kafka.trogdor.workload.ConnectionStressWorker.totalConnectionsSynchronized 66% of the timeUnsynchronized access at ConnectionStressWorker.java:[line 102]Synchronized access at ConnectionStressWorker.java:[line 120]Synchronized access at ConnectionStressWorker.java:[line 120]
IS | Inconsistent synchronization of org.apache.kafka.trogdor.workload.ConnectionStressWorker.totalFailedConnections; locked 50% of time
  | Bug type IS2_INCONSISTENT_SYNC (click for details) In class org.apache.kafka.trogdor.workload.ConnectionStressWorkerField org.apache.kafka.trogdor.workload.ConnectionStressWorker.totalFailedConnectionsSynchronized 50% of the timeUnsynchronized access at ConnectionStressWorker.java:[line 103]Synchronized access at ConnectionStressWorker.java:[line 120]
```
(see IS2_INCONSISTENT_SYNC in http://findbugs.sourceforge.net/bugDescriptions.html)